older-examples/README.md

```diff
@@ -0,0 +1,23 @@
+# Older Examples (Miden Assembly 0.7)
```

Comment:
This is also super useful, let's tackle them one-by-one

Reply:
I opened https://github.com/0xMiden/examples/issues/230